test/HarpoonBattleTest.cpp

```diff
@@ -73,6 +73,33 @@ TEST_F(HarpoonBattleTest, StringToEnum) {
 }
 
 
+TEST_F(HarpoonBattleTest, MergeDataForDifferentTypes) {
+   const std::string uuid = "some-uuid";
+   auto type = KrakenBattle::SendType::Begin;
+   auto error = std::string("no error - ignored");
+   auto data = KrakenIntegrationHelper::GetRandomData(1024);
+   auto merged = KrakenBattle::MergeData(uuid, type, data, error);
+
+   EXPECT_EQ(merged.size(), 1040);  // 1024 + uuid + type
```

Comment:
Can you calculate the 1040 instead?
std::vector<...> expected;
expected . insert uuid
expected . insert KrakenBattle::EnumToString(begin)
expected . insert data_of_1024

EXPECT_EQ(merged.size(), expected.size());